ajax/libs/tracing.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "tracing-js",
```

Comment:
`name` should be the same as your directory name and repo name, which would be `tracing.js`.